Added functions for Deltas

harrysinghtiwana opened the pull request:
Added function getDeltas, getDeltaCursor and createAuthHeader

Commit:
Added functions for Deltas 

Added function getDeltas, getDeltaCursor and createAuthHeader

khamidou commented:
Hi @lawcus. Thanks for the pull request!

This pull request looks good to me; I only have three concerns:
1. It would be nice if `getDeltas` returned a list of `NylasAPIObject`s instead of plain JSON arrays.
2. Could you update `README.md` too?
3. Are you using an IDE? I see you have changed indent style for all the braces in the file. Please revert this.

Also, to merge this I'd need you to sign our [Contributor License Agreement](https://nylas.com/cla.html). It's similar to the one used by meteor and nodejs.

regards,

Karim

Commit:
Fixed the braces as required by Nylas

harrysinghtiwana commented:
We will update it to NylasObject as well as READMe.md in next update. Yes, we are using PHPStorm, removed the braces. Also, we have signed Agreement. Thanks!

Commit:
Removed namespace

Commit:
Merge remote-tracking branch 'nylas/master'

Commit:
Removed namespace for delta cursor and deltas

harrysinghtiwana committed:
Added stream functions to enhance the API.

Commit:
Merge remote-tracking branch 'origin/addedStreamFunctions'

harrysinghtiwana committed:
Added custom written curl function due to Guzzle connectivity issue with Nylas API. It will be improved latter on properly.

Commit:
Merge remote-tracking branch 'origin/addedStreamFunctions'

harrysinghtiwana committed:
Improved get contacts method to return list of filtered emails with name
Added another temporary function to get calendar events results by native curl request.

harrysinghtiwana committed:
Merge branch 'master' of https://github.com/lawcus/nylas-php

Commit:
Added function getRecurringEvents()
